wallet/src/Dscp/Wallet/Config.hs

```diff
@@ -0,0 +1,68 @@
+{-# LANGUAGE OverloadedLabels #-}
+{-# LANGUAGE TypeOperators    #-}
+
+-- | All wallet's configurations.
+
+module Dscp.Wallet.Config
+    ( WalletConfig
+    , WalletConfigRec
+    , HasWalletConfig
+    , defaultWalletConfig
+    , walletConfig
+    , withWalletConfig
+    , fillWalletConfig
+    , witnessUrl
+    , knitCommand
+    , LoggingParams
+
+    , module Dscp.Core.Config
+    ) where
+
+import Control.Lens ((?~))
+import Data.Reflection (Given (..), give)
+import Loot.Config ((:::), (::<), ConfigKind (Final, Partial), ConfigRec, option, sub)
+
+import Dscp.Config
+import Dscp.Core.Config
+import Dscp.Resource.Logging (LoggingParams (..), basicLoggingParams)
+import Dscp.Web
+
+type WalletConfig = CoreConfig ++
+    '[ "wallet" ::<
+       '[ "logging" ::: LoggingParams
+        , "witness" ::: BaseUrl
+        , "knitCommand" ::: Maybe Text
```

Comment:
Arguably, `knitCommand` should not actually be in config, it makes no sense to put it into config so that a wallet always runs a single command on start by default. Probably it's better to parse this separately only from CLI and pass `Maybe Text` value directly to `initializeEverything` as an argument. Especially given that it's only used for `dscp-wallet-cli`, but not for TUI `dscp-wallet`.